### PR TITLE
Aneesa userform

### PR DESCRIPTION
**Pull request by aneesaali**
look at : 
1. src/components/NewProfile.js 
This code has the basic sign up page on react. Includes validation for matching passwords upon submission. Submission button is disabled until terms and conditions is checked.

2. src/components/styles.css
This code is the styling for the field labels and headers. Can be added onto when developing the UI for this page. 
**Question: is this needed or should the styles.js file be used instead?**

### CODE REVIEW

**Review by amywangg — APPROVED** [review]
Very well done, looks like you're grasping react keep it up!

File: src/components/NewProfile.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
+import React, { useCallback, useState } from "react";
+import { Component } from "react";
+import './styles.css'
+
+// Color Palette
```
Comment (vivvvli) [inline src/components/NewProfile.js:5]: I love how you have all the theme's colours and formatting listed here! really makes it easier for future formatting and code adjustments as well

**Review by vivvvli — APPROVED** [review]
Overall great work! So happy you got everything set up and very organized.

**Review by thushv — COMMENTED** [review]
Nice work! Things are organized and easy to read. We can connect this to the database for the next sprint!

File: src/components/NewProfile.js (hunk 1 above)
Comment (aneesaali) [reply src/components/NewProfile.js:5]: that was actually @amywangg, I must give her credit but thank you I will definitely keep in mind to keep colours for future codes

**Pull request merged**